private/buf/cmd/buf/command/generate/generate.go

```diff
@@ -57,71 +57,71 @@ func NewCommand(
 		Short: "Generate stubs for protoc plugins using a template.",
 		Long: `This command uses a template file of the shape:
 
-# The version of the generation template.
-# Required.
-# The valid values are v1beta1, v1.
-version: v1
-# The plugins to run. "plugin" is required.
-plugins:
-    # The name of the plugin.
-    # By default, buf generate will look for a binary named protoc-gen-NAME on your $PATH.
-    # Alternatively, use a remote plugin:
-    # plugin: buf.build/protocolbuffers/go:v1.28.1
-  - plugin: go
-    # The the relative output directory.
-    # Required.
-    out: gen/go
-    # Any options to provide to the plugin.
-    # This can be either a single string or a list of strings.
-    # Optional.
-    opt: paths=source_relative
-    # The custom path to the plugin binary, if not protoc-gen-NAME on your $PATH.
-    # Optional, and exclusive with "remote".
-    path: custom-gen-go
-    # The generation strategy to use. There are two options:
-    #
-    # 1. "directory"
-    #
-    #   This will result in buf splitting the input files by directory, and making separate plugin
-    #   invocations in parallel. This is roughly the concurrent equivalent of:
-    #
-    #     for dir in $(find . -name '*.proto' -print0 | xargs -0 -n1 dirname | sort | uniq); do
-    #       protoc -I . $(find "${dir}" -name '*.proto')
-    #     done
-    #
-    #   Almost every Protobuf plugin either requires this, or works with this,
-    #   and this is the recommended and default value.
-    #
-    # 2. "all"
-    #
-    #   This will result in buf making a single plugin invocation with all input files.
-    #   This is roughly the equivalent of:
-    #
-    #     protoc -I . $(find . -name '*.proto')
-    #
-    #   This is needed for certain plugins that expect all files to be given at once.
-    #
-    # If omitted, "directory" is used. Most users should not need to set this option.
-    # Optional.
-    strategy: directory
-  - plugin: java
-    out: gen/java
-    # Use the plugin hosted at buf.build/protocolbuffers/python at version v21.9.
-    # If version is omitted, uses the latest version of the plugin.
-  - plugin: buf.build/protocolbuffers/python:v21.9
-    out: gen/python
+	# The version of the generation template.
+	# Required.
+	# The valid values are v1beta1, v1.
+	version: v1
+	# The plugins to run. "plugin" is required.
+	plugins:
+		# The name of the plugin.
+		# By default, buf generate will look for a binary named protoc-gen-NAME on your $PATH.
+		# Alternatively, use a remote plugin:
+		# plugin: buf.build/protocolbuffers/go:v1.28.1
+	  - plugin: go
+		# The the relative output directory.
+		# Required.
+		out: gen/go
```

Comment:
I'd leave the space formatting alone. Tabs are illegal in YAM. It'll fail if the user tries to copy/paste it.